README.md

```diff
@@ -106,6 +106,13 @@ Other things worth noting:
     - Unlike in lifxlan, set_waveform takes a dictionary with the right
       keys instead of all those parameters
 
+# Development
+To develop locally, the easiest way is to from the root directory run:
+
+```bash
+pip3 install . && python3 ./examples/lifx-cli.py
```

Comment:
```suggestion
pip3 install -e . && python3 ./examples/lifx-cli.py
```

Reply:
Well.... from the project root directory

     python3 -m aiolifx

is a lot more convenient.  Clearly you need to have the dependencies installed, but I usually have the latest stable  version installed on the system I use.

The file examples/lifx-cli.py was created before I decided to have a CLI utility.